rwgame/GameInput.cpp

```diff
@@ -62,19 +62,19 @@ const std::unordered_multimap<int, GameInputState::Control> kDefaultControls = {
     {SDLK_e, GameInputState::LookRight},
 };
 
-void GameInput::updateGameInputState(GameInputState *state,
-                                     const SDL_Event &event) {
+void GameInput::updateGameInputState(
+    GameInputState *state, const SDL_Event &event) {
     switch (event.type) {
-        case SDL_KEYDOWN:
-        case SDL_KEYUP: {
-            auto sym = event.key.keysym.sym;
-            auto level = event.type == SDL_KEYDOWN ? 1.f : 0.f;
-            auto& levels = state->levels;
+    case SDL_KEYDOWN:
+    case SDL_KEYUP: {
+        auto sym = event.key.keysym.sym;
+        auto level = event.type == SDL_KEYDOWN ? 1.f : 0.f;
+        auto &levels = state->levels;
```

Comment:
Again, apparently same goes for references. It happens in other places too, too much to comment everything.